office/timewarrior/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.0
```

Comment:
Usually we want to use the latest version of a portgroup, which here is cmake 1.1. If you change that, then the line `cmake.out_of_source yes` can be deleted, because that is the default of cmake 1.1.

Reply:
Good point; fixed.